granulate_utils/linux/cgroups/cpu_cgroup.py

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from granulate_utils.linux.cgroups.base_cgroup import BaseCgroup
+
+
+class CpuCgroup(BaseCgroup):
+    HIERARCHY = "cpu"
+
+    def set_cpu_limit_cores(self, cores: float) -> None:
+        period = int(self.read_from_controller("cpu.cfs_period_us").split("\n")[0])
```

Comment:
```suggestion
        period = int(self.read_from_controller("cpu.cfs_period_us"))
```
it'll just work, `int` can handle whitespaces around the number. Same fix in other sites.